app/actions/CommonActions.scala

```diff
@@ -27,4 +29,20 @@ object CommonActions {
       }
     }
   }
+
+  case class ABTestRequest[A](testId: Int, request: Request[A]) extends WrappedRequest(request) {
```

Comment:
Am I right to assume `testId` is a user related id rather than a test related id?